src/main/resources/db/migration/V5__auto_accept_campaigns.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE `campaigns` ADD COLUMN
+`auto_accept` BOOL NOT NULL DEFAULT TRUE
+;
+
+ALTER TABLE `device_updates` MODIFY COLUMN
+`status` ENUM('scheduled', 'accepted', 'successful', 'cancelled', 'failed') NOT NULL
```

Comment:
I think, this will lead to an API endpoint of createCampaign(name, desc, ... , autoAccept = TRUE).

IMO, as API description, it is good to have an endpoint like createCampaign(name, desc, ... , approvalNeeded = FALSE) so that consumer of this API (for example: webapp) can call with approvalNeeded = TRUE/FALSE accordingly. approvalNeeded will be FALSE by default.

Reply:
changed